backend/src/main/java/com/llm_service/llm_service/service/LLMService.java

```diff
@@ -14,7 +14,7 @@ public class LLMService {
     final String suffix = "\n    return result\n";
 
     public LLMService() {
-        modelParameters = new ModelParameters().setModelFilePath("model/llama-2-13b-chat.Q5_K_S.gguf");
+        modelParameters = new ModelParameters().setModelFilePath("model/llama-2-13b-chat.Q5_K_M.gguf");
```

Comment:
revert this change, since the other modal was the decided one , please check the readme